_tasks/test_browser/server.ts

```diff
@@ -0,0 +1,15 @@
+import { serveDir } from "https://deno.land/std@0.172.0/http/file_server.ts"
+import { serve } from "https://deno.land/std@0.172.0/http/server.ts"
+
+import { PORT, STATIC_DIR } from "./constants.ts"
+
+const ac = new AbortController()
+
+const handler = async (req: Request): Promise<Response> =>
```

Comment:
Not sure why you're splitting apart so many blocks. You can inline the handler in `serve`, given that it's not utilized elsewhere. Same thing with the `() => ac.abort()`.  Frankly, same thing with the constants file and others. There's not a ton of code here. Let's git it all in a single `_tasks/test_browser.ts`.